core/preconditioner/jacobi.cpp

```diff
@@ -50,6 +54,44 @@ GKO_REGISTER_OPERATION(initialize_precisions, jacobi::initialize_precisions);
 }  // namespace jacobi
 
 
+template <typename ValueType, typename IndexType>
+typename Jacobi<ValueType, IndexType>::parameters_type
+Jacobi<ValueType, IndexType>::parse(const config::pnode& config,
+                                    const config::registry& context,
+                                    const config::type_descriptor& td_for_child)
+{
+    auto params = preconditioner::Jacobi<ValueType, IndexType>::build();
+
+    if (auto& obj = config.get("max_block_size")) {
+        params.with_max_block_size(gko::config::get_value<uint32>(obj));
+    }
+    if (auto& obj = config.get("max_block_stride")) {
+        params.with_max_block_stride(gko::config::get_value<uint32>(obj));
+    }
+    if (auto& obj = config.get("skip_sorting")) {
+        params.with_skip_sorting(gko::config::get_value<bool>(obj));
+    }
+    // SET_VALUE_ARRAY(factory, gko::array<IndexType>, block_pointers,
```

Comment:
I would recommend not supporting these kind of parameters, see my comment in #1480